data/hai/hai missions.txt

```diff
@@ -1932,75 +1932,101 @@ mission "Station Exploration Abandon Ship"
 			`	"You are being fined <fine> for abandoning your passengers without warning. You're lucky you didn't end up with a more serious punishment." After further chewing you out for endangering your passengers' safety, he walks away from you in a huff.`
 
-mission "Pilgrimage"
+mission "Hai Pilgrimage"
 	name "Hai spiritual leader to <planet>"
-	description "Bring a Hai spiritual leader to <destination>."
+	description "Bring a Hai spiritual leader to <destination>. He will pay you <payment>."
 	minor
+	clearance "You inform the authorities of the passenger you are transporting, and they quickly allow you to land."
 	source
 		government "Hai"
-		not attributes "station"
+		not attributes "uninhabited" "station"
+		not planet "Darkmetal" "Dustmaker" "Stonebreak" "Skyfarm"
 	destination
 		attributes "unfettered"
 	passengers 1
 	to offer
-		random < 30
-		has "outfit: Quantum Keystone"
+		random < 10
+		or
+			has "outfit: Quantum Keystone"
+			has "outfit: Quantum Key Stone"
+			has "outfit: Shield Beetle Pendant"
+			has "outfit: Tree Skeleton Key Stone"
 		has "First Contact: Hai: offered"
 	on offer
 		conversation
-			`It is an unusually foggy day on <source>. The air is so thick with moisture that you cannot see more than a few paces in front of you. Perhaps that is why you are so startled when you turn around and see a Hai standing right next to you. He is so close you can see dewdrops on his whiskers, and you immediately notice the prominent jewelry that he is wearing. He appears to be quite old, judging by his stooped-over posture and the white fur around his muzzle.`
-			`	"I see that you are wiser than most humans here," he says, pointing towards your ship. "You carry a Keystone, a shield against the tempest of space. Let no one else tell you otherwise. Are you willing and able to take me to <destination>? I can pay <payment>."`
+			`It is an unusually foggy day on <origin>; the air is so thick with moisture that you cannot see more than a few paces in front of you. Perhaps that's why you're so startled when you turn around and see a Hai standing right next to you. He is so close you can see dewdrops on his whiskers, and you immediately notice the prominent jewelry that he is wearing. He appears to be quite old, judging by his stooped-over posture and the white fur around his muzzle.`
+			branch pendant
+				has "outfit: Shield Beetle Pendant"
+			branch normal
+				has "outfit: Quantum Keystone"
+			branch remnant
+				has "outfit: Quantum Key Stone"
+			branch huh
+				has "outfit: Tree Skeleton Key Stone"
+			label normal
+			`	"I see that you are wiser than most humans here," he says, pointing towards your ship. Its hatch is open, prominently displaying your Keystone to any potential travellers. "You carry a shield against the tempest of space. Let no one else tell you otherwise." He stops to catch his breath. "Now, onto more worldly things. My name is Hareek. Are you willing and able to take me to <destination> for <payment>?"`
```

Comment:
```suggestion
			`	"I see that you are wiser than most humans here," he says, pointing toward your ship. Its hatch is open, prominently displaying your keystone to any potential travelers. "You carry a shield against the tempest of space. Let no one else tell you otherwise." He pauses to take a breath. "Now, onto more worldly things. My name is Hareek. Are you willing and able to take me to <destination> for <payment>?"`
```
I think it makes more sense to make him just pause to breathe rather than play up the fact that he's winded.